blacklight-cornell/app/views/browse/_heading.html.erb

```diff
@@ -46,13 +46,13 @@
             <% if !data["seeAlso"].nil? || !data["notes"].nil?  || data["mainEntry"] == true %>
               <% unless params[:browse_type] == 'Author-Title' %>
 				<% if %>
-                  <%= link_to '/browse/info?authq=' + encoded_heading +'&browse_type=' + params[:browse_type]+"&headingtype=" + data["headingTypeDesc"], { :class => "info-button btn btn-sm btn-outline-secondary", :onclick => "javascript:_paq.push(['trackEvent', 'browse', 'info-btn']);"} do %>
+                  <%= link_to '/browse/info?authq=' + encoded_heading +'&browse_type=' + params[:browse_type]+"&headingtype=" + data["headingTypeDesc"], { :class => "info-button btn btn-xs btn-outline-secondary", :onclick => "javascript:_paq.push(['trackEvent', 'browse', 'info-btn']);", 'aria-label' => {params[:browse_type]} + ' info for ' +  data["heading"]} do %>
```

Comment:
This needs to either be: `'aria-label' => params[:browse_type] + ' info for ' + data["heading"]` (no curly brackets around `params[:browse_type]` or `'aria-label' => "#{params[:browse_type]} info for #{data['heading"]}"`(string interpolation). I believe the tests are failing because the subject and author browse index pages are breaking.